File: 백준/Bronze/2908. 상수/상수.py
import sys

input = sys.stdin.readline
ans = list(map(list, input().split()))
tmp = ""

for i in range(0, 2):
    tmp = ans[i][0]
    ans[i][0] = ans[i][2]
    ans[i][2] = tmp
    ans[i] = int("".join(ans[i]))

print(ans[0]) if ans[0] > ans[1] else print(ans[1])

File: 백준/Bronze/2908. 상수/상수.js
const fs = require("fs");
const filepath = process.platform === "linux" ? 0 : "run/input.txt";
const input = fs.readFileSync(filepath, "utf-8").split("\n");

let arr = input[0].split(" ");
let tmp;

for (let i = 0; i < 2; i++) {
  arr[i] = arr[i].split("");
  tmp = arr[i][0];
  arr[i][0] = arr[i][2];
  arr[i][2] = tmp;

  arr[i] = Number(arr[i].join(""));
}

console.log(arr[0] > arr[1] ? arr[0] : arr[1]);
